### PR TITLE
chore: fix invalid symbol '"'

### PR DESCRIPTION
**Pull request by paulbalandan**
**Description**
This fixes a warning given by the Github Actions extension on vscode.
image

**Checklist:**
- [x] Securely signed commits
- [ ] Component(s) with PHPDoc blocks, only if necessary or adds value
- [ ] Unit testing, with >80% coverage
- [ ] User guide updated
- [ ] Conforms to style guide

### CODE REVIEW

**Review by michalsn — APPROVED** [review]
We can make the same fixes to the devkit.

**Comment by paulbalandan** [comment]
Done, see codeigniter4/devkit#132

**Comment by michalsn** [comment]
Thank you!

**Comment by paulbalandan** [comment]
See https://github.com/actions/runner/issues/753#issuecomment-1602902095 for the proper fix.

**Comment by michalsn** [comment]
Well, the syntax is not very intuitive, but it works, so 👍

**Review by ddevsr — APPROVED** [review]

File: .github/workflows/test-psalm.yml [hunk 1]
```diff
@@ -24,7 +24,7 @@ jobs:
   build:
     name: Psalm Analysis
     runs-on: ubuntu-latest
-    if: "!contains(github.event.head_commit.message, '[ci skip]')"
+    if: '! contains(github.event.head_commit.message, ''[ci skip]'')'
```
Comment (kenjis) [inline .github/workflows/test-psalm.yml:27]: PhpStorm reports an error. Is this false positive?
Screenshot 2024-07-23 13 26 36

Comment (paulbalandan) [reply .github/workflows/test-psalm.yml:27]: Hmm, it seems we cannot satisfy all worlds.

Could you try this change:
```diff
diff --git a/.github/workflows/test-psalm.yml b/.github/workflows/test-psalm.yml
index b376416e27..145d984b83 100644
--- a/.github/workflows/test-psalm.yml
+++ b/.github/workflows/test-psalm.yml
@@ -24,7 +24,7 @@ jobs:
   build:
     name: Psalm Analysis
     runs-on: ubuntu-latest
-    if: '! contains(github.event.head_commit.message, ''[ci skip]'')'
+    if: (! contains(github.event.head_commit.message, '[ci skip]'))
 
     steps:
       - name: Checkout
```

Comment (kenjis) [reply .github/workflows/test-psalm.yml:27]: PhpStorm does not complain about it.

Comment (paulbalandan) [reply .github/workflows/test-psalm.yml]: Resorted to using parentheses instead.

**Review by kenjis — APPROVED** [review]

**Comment by kenjis** [comment]
@paulbalandan Can you update https://github.com/codeigniter4/.github ?

**Comment by MGatner** [comment]
Thanks for this

**Comment by kenjis** [comment]
I sent https://github.com/codeigniter4/.github/pull/27

**Pull request merged**